lib/rouge/lexers/python.rb

```diff
@@ -68,7 +68,7 @@ def self.exceptions
         end
 
         rule /[^\S\n]+/, Text
-        rule /#.*$/, Comment
+        rule %r(#(.*)?\n), Comment::Single
```

Comment:
I think we should add a `?` after the `\n` so it's optional to account for the rare case of a comment being on the last line of a file without a newline. It's not super likely to happen but it worked prior to this change so we should preserve that.

Reply:
Fine by me, I've updated the PR